### DISCUSSION TITLE
Provide Kubernetes deployment support

### DISCUSSION THREAD

**Discussion by MohammedFadin** [specify/specify7#5742] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Since Docker compose is the recommended approach of installation, does that mean Kubernetes is not supported? 

**Describe the solution you'd like**
To support Kubernetes and other orchestration platforms. Docker Compose is not a good solution in my opinion as it lacks the Orchestration capabilities

**Reported By**
Personal

**Comment by maxpatiiuk** [comment]
Hi. Thanks for the question.

At the moment, Specify 7 is not available as a Kubernetes deployment. Unfortunately, none of the core Specify team members have much experience with Kubernetes so we can't even tell how hard it would be to make it work in Kubernetes.

However, if you are interested in assisting us in converting the docker-compose to Kubernetes, we would be happy to provide information on the current architecture and how different containers interact.

In the meanwhile, can you take a look at the Docker Stack to see if that has sufficient Orchestration capabilities for your needs?
